Gemfile

```diff
@@ -8,6 +8,10 @@ gem 'rails', '4.1.9'
 # use postgres database
 gem 'pg'
 
+group :development, :test do
+  gem 'sqlite3'
```

Comment:
Why not use postgres everywhere? I don't think it's worth adding another dependency.